### ISSUE TITLE
add "incident price change" column to orders export

### ISSUE THREAD

**Issue by Paul-Eraman-CoopCycle** [coopcycle/coopcycle-web#4587]
In orders export, the new price based on a reported incident is shown, but you cant see it was changed. propsal:

- [ ] new column "incident price change" : shows the +/- amount changed by the incident
- [ ] new column "incidents" : shows the name of the reported incident(s) attached to an order

**Comment by r0xsh** [comment]
>  new column "incidents" : shows the name of the reported incident(s) attached to an order

Regarding the order-to-incident relationship mapping in the CSV export, we have two possible approaches:

1. Create a separate CSV file specifically for incidents, maintaining the one-to-many relationship between orders and their associated incidents.

2. Include all incidents within the order rows, where each order row would contain multiple incident entries (potentially making the row structure more complex).

Each approach has its trade-offs:
- Separate CSVs: Cleaner data structure but requires handling multiple files
- Combined format: Single file but more complex row structure

IMHO I'll go for the first approach since it's way easier to manipulate the data like so

**Comment by Paul-Eraman-CoopCycle** [comment]
Then let's go with multiple exports

However, for users could we have it be one button that exports one excel with two tabs?

**Comment by Paul-Eraman-CoopCycle** [comment]
Maybe the total incident price change and final price should be in the universal excel even if the list of incidents needs to be on a separate excel?

**Comment by r0xsh** [comment]
Yup, already added the incident price change + final price in the universal excel, the issue what with the incident titles

**Comment by Paul-Eraman-CoopCycle** [comment]
Beautiful :)
